frontend/src/metabase/containers/CreateCollectionOnTheGo.tsx

```diff
@@ -0,0 +1,66 @@
+import { useState, createContext, useCallback, ReactElement } from "react";
+import { t } from "ttag";
+import { Collection, CollectionId } from "metabase-types/api";
+import CreateCollectionModal from "metabase/collections/containers/CreateCollectionModal";
+import { NewCollectionButton } from "./CreateCollectionOnTheGo.styled";
+
+interface State {
+  enabled: boolean;
+  resumedValues: any;
+  openCollectionId?: CollectionId;
+}
+
+interface Props {
+  children: (resumedValues: any) => ReactElement;
+}
+
+interface ButtonProps {
+  resumedValues: any;
```

Comment:
It would be great to avoid `any`

Reply:
Ack.  I will look at this tomorrow.

Reply:
3b3bbb3e95304b7663f6ec87e72801b97e72ad6d